biome.base.json

```diff
@@ -0,0 +1,40 @@
+{
+  "$schema": "https://biomejs.dev/schemas/1.9.4/schema.json",
+  "vcs": {
+    "enabled": false,
+    "clientKind": "git",
+    "useIgnoreFile": true
+  },
+  "files": {
+    "ignoreUnknown": false,
+    "ignore": [
+      "tsconfig.json",
+      "tsconfig.*.json",
```

Comment:
Digging into [docs](https://biomejs.dev/internals/language-support/#jsonc-support) looks like they have specific rules to handle tsconfig.json files, I think the problem we are running into is that it doesn't recognize `tsconfig.*.json` files as having the same exception. We could also solve this by adding our own overrides, like so:

```json
 "overrides": [
    {
      "include": ["tsconfig.*.json"],
      "json": {
        "parser": {
          "allowComments": true,
          "allowTrailingCommas": true
        },
        "formatter": {
          "trailingCommas": "none"
        }
      }
    }
  ]
``` 

This should allow us to match the expected behaviour on our custom tsconfig.*.json files without having to ignore them completely. WDT?

Reply:
Yeah probably not. It's good to know, if not just so we know how to handle it either way. I'm good with ignoring the tsconfig.*.json